src/bp/core/services/dialog/dialog-engine.ts

```diff
@@ -268,11 +268,17 @@ export class DialogEngine {
         return event
       }
 
+      const executeParentNode = transitionTo.startsWith('##')
```

Comment:
Well technically you can name it #Something, to go there and execute the node. I highly doubt someone already named his nodes with # since it's a special keyword.

Reply:
Well no one knows that except us hehe